docs/install/dataplane-service-account.md

```diff
@@ -28,11 +28,25 @@ In general, we would just need permissions to receive messages
 (`roles/pubsub.subscriber`). However, in the case of the `Channel`, we would
 also need the ability to publish messages (`roles/pubsub.publisher`).
 
-1. Create a new Service Account named `cre-dataplane` with the following
-   command:
+1. Create a new Google Cloud Service Account (GSA) named `events-sources-gsa`
+   with the following command:
 
    ```shell
-   gcloud iam service-accounts create cre-dataplane
+   gcloud iam service-accounts create events-sources-gsa
+   ```
+
+   Depending on the use case, you may want to have a single GSA for all the
+   sources, like the example above, or setup multiple similar service accounts.
+   If you are setting up multiple GSAs you can follow the same set of steps, but
+   take note to configure authentication only for the namespaces that are
+   intended for each account.
+
+   Additionally, while it is possible to use the same GSA for both the broker
+   and the sources, we recommend creating a dedicated Google Service Account for
+   the broker data plane (e.g. `events-broker-gsa`):
+
+   ```shell
+   gcloud iam service-accounts create events-sources-gsa
```

Comment:
This looks identical to the previous shell command, should it be for `events-broker-gsa`?

Reply:
Fixed.